acceptance/bundle/resources/secret_scopes/basic/script

```diff
@@ -0,0 +1,39 @@
+export SECRET_SCOPE_NAME="test-scope-$UNIQUE_NAME-1"
+
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+    rm out.requests.txt
+}
+trap cleanup EXIT
+
+title "create the secret scope"
+trace $CLI bundle plan
+trace $CLI bundle deploy
+
+scope_name=$($CLI bundle summary --output json | jq -r '.resources.secret_scopes.my_scope.name')
+trace $CLI secrets list-scopes -o json | jq ".[] | select(.name == \"$scope_name\")"
+
+title "put and get secret in first scope"
+trace $CLI secrets put-secret $scope_name my-key --string-value "my-secret-value"
+trace $CLI secrets get-secret $scope_name my-key
+
+trace print_requests.py //secrets
+
+title "update the name of the scope (should recreate)"
+export SECRET_SCOPE_NAME="test-scope-$UNIQUE_NAME-2"
+envsubst < databricks.yml.tmpl > databricks.yml
+
+trace $CLI bundle plan
+trace $CLI bundle deploy
+
+scope_name=$($CLI bundle summary --output json | jq -r '.resources.secret_scopes.my_scope.name')
+trace $CLI secrets list-scopes -o json | jq ".[] | select(.name == \"$scope_name\")"
+
+title "put and get secret in recreated scope"
+trace $CLI secrets put-secret $scope_name another-key --string-value "another-secret-value"
+trace $CLI secrets get-secret $scope_name another-key
+
+# Capture API requests for verification. Terraform cleans up ACLs before deleting the scope, but direct does not, hence the difference in requests.
+trace print_requests.py //secrets > out.second-requests.$DATABRICKS_BUNDLE_ENGINE.txt
```

Comment:
"second-requests" implies there is a "first" one. Maybe name "recreate"?

Reply:
And it can be done earlier, right after the recreate, to make it smaller.

It now includes the `put-secret` call.